runsc/boot/loader.go

```diff
@@ -942,7 +942,11 @@ func (l *Loader) createContainerProcess(cid string, info *containerInfo) (*kerne
 	if len(info.goferFDs) < 1 {
 		return nil, nil, fmt.Errorf("rootfs gofer FD not found")
 	}
-	l.startGoferMonitor(cid, int32(info.goferFDs[0].FD()))
+	// TODO: The gofer monitor should be started as long as this container has a gofer
```

Comment:
nit: can you actually assign this TODO to me `TODO(ayushranjan)`. I need to consolidate this logic once #9624 goes in. Because once that goes in, we should always have a gofer FD to monitor the gofer if a gofer process exists.

Reply:
Sure, thanks!